06a-WGS.Rmd

```diff
@@ -0,0 +1,118 @@
+
+These are some of the commonly used copy number variation analysis tools: 
+- [GATK](https://gatk.broadinstitute.org/hc/en-us/articles/360035531092--How-to-part-I-Sensitively-detect-copy-ratio-alterations-and-allelic-segments) GATK has a variety of tools that ca nbe used to study changes in copy numbers of genes. This link provides a tutorial for how to use the tools. 
+- [AscatNGS](https://www.crick.ac.uk/research/labs/peter-van-loo/software) This tools (allele-specific copy number analysis of tumors) is specific for WGS copy number variation analysis. It can be used to dissect allele-specific copy numbers of tumors by estimating and adjusting for tumor ploidy and nonaberrant cell admixture.  
+- [TitanCNA](https://github.com/gavinha/TitanCNA) This tool, developed by Fred Hutch Principal Investigator Dr. Gavin Ha, is used to analyze copy number variation and loss of heterozygoisity at the subclonal level for both WGS and WES data in tumors compared to matched normals. This tool accounts for mixtures of cell populations and estimates the proportion of cells harboring each event. The Ha lab has developed a [snakemake pipeline](https://github.com/gavinha/TitanCNA/tree/master/scripts/snakemake) to more easily use this tool.   
+- [gGNV](https://gatk.broadinstitute.org/hc/en-us/articles/4404604788891-GermlineCNVCaller) This is a germline CNV calling tool that can be used on both WGS and WES data. This tool has booth COHORT and CASE modes. COHORT mode is used when providing a cohort of germline samples where CASE mode is used for individual samples. 
+- [BIC-seq2](http://compbio.med.harvard.edu/BIC-seq/) This tool i used to detect CNVs with or without control samples. The steps involved in this data processing tool include normalization and CNV detection. 
+
+Here are the tools frequently used to visualize the output data from the above tools: 
+- [IGV](https://software.broadinstitute.org/software/igv/) IGV is an interactive tool used to easily visualize genomic data. It is available as a desktop application, web application, and JavaScript to embed in web pages. This application is very beneficial for visualizing both mutational and CNV data for WGS. IGV has many [tutorials](https://www.youtube.com/channel/UCb5W5WqauDOwubZHb-IA_rA/featured) on youtube that are helpful for using the tool to its full potential. 
+- [Maftools](https://www.bioconductor.org/packages/devel/bioc/vignettes/maftools/inst/doc/maftools.html) Maftools is an R package that can be used to create informative plots from your WGS data output. It has tools to import both VCF files and ANNOVAR output for data analysis. 
+- [Prism](https://www.graphpad.com/scientific-software/prism/) Prism is a widely used tool in scientific research for organizing large datasets, generating plots, and creating readable figures. WGS data regarding mutations and CNV can be used as input for creating plots with this tool. 
```

Comment:
This mention might get moved to a different chapter since it's not really specific to sequencing data. But we can leave it here for now. My plan is to have a chapter that generally discusses data analysis tools folks may want to consider. But we don't have that chapter up and running yet.

Reply:
that sounds good!